db/db.connect.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose')
+mongoose.plugin(require('meanie-mongoose-to-json'));
+const Secret = process.env['secret_key']
+
+async function connectDB() {
+  try {
+    await mongoose.connect(`mongodb+srv://Kunal:${Secret}@neog-cluster.epsoe.mongodb.net/videolibrary?retryWrites=true&w=majority`, { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: false })
```

Comment:
this whole URI can be moved to the env variable as the bane is still visible here